examples/dataservice-example/CMakeLists.txt

```
@@ -15,37 +15,41 @@
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
 
-set(OLP_SDK_DATASERVICE_WRITE_EXAMPLE_TARGET dataservice-write-example)
+set(OLP_SDK_DATASERVICE_EXAMPLE_TARGET dataservice-example)
 set(OLP_SDK_EXAMPLE_SUCCESS_STRING "Publish Successfull")
```

```suggestion
set(OLP_SDK_EXAMPLE_SUCCESS_STRING "Example is successfully finished")
```
This string was displaying for the user when the example finished execution. Before your changes, it displayed different message depending on the example being run. 

So, now, it is needed to change the message in more general way... Maybe in future we will create richer UI